keyboards/butterstick/butterstick.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "quantum.h"
+
+#define BUTTER_LAYOUT( 										\
```

Comment:
lower case descriptions and the other way around is the format here

```suggestion
#define LAYOUT_butter( 										\
```

Reply:
Fixed in https://github.com/qmk/qmk_firmware/pull/5742/commits/ed263ea00aa379ffd144ad9219f682cddeaadda0